server/lib/rfidAdafruitPN532ShieldConnection.js

```diff
@@ -0,0 +1,116 @@
+/** hex-reader.ino 
```

Comment:
Added more of an explanation - the only NFC reader I had on hand was an Arduino shield, so i made a quick sketch to use the Arduino as a glorified SPI-to-usbserial adapter. It would be nice to just directly connect it to the raspberry pi, which i might do in the future after writing up a small blog post about the whole thing.